### PR TITLE
Expand integration overview page

### PR DESCRIPTION
**Pull request by katomm**
Expand integration overview page with various links and components

### CODE REVIEW

**Review by rphair — APPROVED** [review]
very good; nice also to have all the canonical repository links in one place after their shuffling around 🤓

File: docs/integrate-cardano/overview.md [hunk 1]
```diff
@@ -13,9 +13,24 @@ image: /img/og/og-developer-portal.png
 ## Introduction
 Here we show you how to integrate Cardano into existing websites and services. 
 
-## Topics
+## Integration Components
+- [Overview](/docs/get-started/cardano-components.md) of the different Cardano commponents.
+- [cardano-node](https://github.com/input-output-hk/cardano-node) is the top level for the node and aggregates the other components from other packages: consensus, ledger and networking, with configuration, CLI, logging and monitoring.
+- [cardano-wallet](https://github.com/cardano-foundation/cardano-wallet) helps you manage ada. You can use it to send and receive payments on the Cardano blockchain via a http and cli interface.
```
Comment (rdlrt) [inline docs/integrate-cardano/overview.md:19]: Not sure if cardano-wallet is a core component anymore

**Review by rdlrt — APPROVED** [review]
Unsure about cardano-wallet, but otherwise LGTM

**Comment by rphair** [comment]
I still use it as if it were a core component 🤓

**Review by fill-the-fill — APPROVED** [review]

File: docs/integrate-cardano/overview.md (hunk 1 above)
Comment (fill-the-fill) [reply docs/integrate-cardano/overview.md:19]: Is this resolved?

**Review by os11k — APPROVED** [review]

File: docs/integrate-cardano/overview.md (hunk 1 above)
Comment (rphair) [reply docs/integrate-cardano/overview.md:19]: I think this tiebreaker should be that we have a component page still dedicated to it here on the Portal 🤓 _Get Started > Cardano Components > Installing cardano-wallet_

**Pull request merged**